.travis.yml

```diff
@@ -12,7 +12,7 @@ env:
                 - UPDATE_TOTAL="$(find test/functional/update -name *.bats | wc -l)"
                 - UPDATE_SUBGROUP1_TOTAL=17
                 - UPDATE_SUBGROUP2_TOTAL="$((UPDATE_TOTAL - UPDATE_SUBGROUP1_TOTAL))"
-                - UPDATE_SUBGROUP1="$(find test/functional/update -name *.bats | head -n $UPDATE_SUBGROUP1_TOTAL | tr '\n' ' ')"
+                - UPDATE_SUBGROUP1="$(find test/functional/{autoupdate, update} -name *.bats | head -n $UPDATE_SUBGROUP1_TOTAL | tr '\n' ' ')"
```

Comment:
don't add these here, you need to move this in a group not a subgroup

Reply:
if you check your Travis execution you will notice about 25 tests were not run because of this change

Reply:
Yikes, Yes..I have an extra space. My bad

Reply:
Done

Reply:
please don't add autoupdate to UPDATE_SUBGROUP1 that is ment for update related tests, you want the run as part of worker 1 so add autoupdate to GROUP1

Reply:
Yes corrected it